.github/workflows/image-check.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
```

Comment:
since we're editting these files i think we should converge on the same action version. (ie functests and unittests are using `actions/setup-go@v4`, but image-check is v2 and kind-integration is v1)

Reply:
Not sure what these action versions signify tbh. What version should we converge on? And should it be the same version across all actions? As in, could it be different for specific actions - say checkout set to v2 and setup set to v4, or should we have just one version?